planner/core/foreign_key.go

```diff
@@ -45,12 +45,13 @@ type FKCheck struct {
 // FKCascade indicates the foreign key constraint cascade behaviour.
 type FKCascade struct {
 	basePhysicalPlan
-	Tp         FKCascadeType
-	ReferredFK *model.ReferredFKInfo
-	ChildTable table.Table
-	FK         *model.FKInfo
-	FKCols     []*model.ColumnInfo
-	FKIdx      *model.IndexInfo
+	Tp           FKCascadeType
+	ReferredFK   *model.ReferredFKInfo
+	ChildTable   table.Table
+	FK           *model.FKInfo
+	FKCols       []*model.ColumnInfo
+	FKIdx        *model.IndexInfo
+	CascadePlans []Plan
```

Comment:
So, the `CascadePlans` are seen as the "children" of the `FKCascade`.
But it's filled during "executing" instead of "building plan", which also means we'll see different plan in `EXPLAIN` and `EXPLAIN ANALYZE`.
I think we need some comments to clarify this.